src/main/java/io/github/fabriccommunity/everything/storage/AbstractThing.java

```diff
@@ -0,0 +1,27 @@
+package io.github.fabriccommunity.everything.storage;
+
+// Stores a thing, but it's abstract now.
+public abstract class AbstractThing<T> {
```

Comment:
Should be `ThingAbstract`

Reply:
Should be `Abstract`

Reply:
Should be `feb5f4784065401e896fe5e8055125f5`

Reply:
did the thing